PullToRefresh/PullToRefresh.swift

```diff
@@ -11,13 +11,13 @@ import Foundation
 
 public protocol RefreshViewAnimator {
      func animateState(state: State)
+    func animateLoading(animations animations: () -> (), completion: (Bool) -> ())
+    func animateFinished(animations animations: () -> (), completion: (Bool) -> ())
```

Comment:
It's a bit confusion because you can call two methods for finished\loading state:
1. animateState with finished state
2. animateFinished
But they have different implementations